app/models/user.rb

```diff
@@ -256,14 +252,6 @@ def destroy_widget_sets_for_group(group_id)
     subscribed_widget_sets_for_group(group_id).destroy_all
   end
 
-  def destroy_orphaned_dashboards
-    (group_ids_of_subscribed_widget_sets - miq_group_ids).each { |group_id| destroy_widget_sets_for_group(group_id) }
```

Comment:
Are `destroy_widget_sets_for_group` and `group_ids_of_subscribed_widget_sets` (and other methods added for 30cee4e7) needed anymore?

@lfu Can you review this one?